.github/workflows/assignproj.yml

```diff
@@ -4,10 +4,6 @@ on:
   issues:
     types: [opened]
 
-permissions:
-  issues: write
```

Comment:
I don't think we actually need this permission set. Was leftover from a previous attempt.